network/robus/inc/config.h

```diff
@@ -13,12 +13,15 @@
 #define DEFAULTID         0x00
 #define PROTOCOL_REVISION 0
 #define BROADCAST_VAL     0x0FFF
-#define DEFAULTBAUDRATE   1000000
 
 #define TIMEOUT_VAL       2
 #define MAX_ALIAS_SIZE    16
 #define MAX_DATA_MSG_SIZE 128
 
+#ifndef DEFAULTBAUDRATE
+#define DEFAULTBAUDRATE 1000000
```

Comment:
Watch out your commit comment "make default bauderate changeable" should be "Make default bauderate configurable"
Also this configuration should appear on every `node_config.h` of every projects...